apps/web/hooks/use-escrows.ts

```
@@ -203,6 +204,7 @@ export const useEscrowsBySignerQuery = ({
 export function useCreateEscrow(onSuccessCallback?: () => void) {
   const queryClient = useQueryClient();
   const router = useRouter();
+  const { address } = useGlobalAuthenticationStore();
```

Biome (2.4.9)

[error] This variable address is unused.

(lint/correctness/noUnusedVariables)